coord/src/module.c

```diff
@@ -2459,7 +2449,8 @@ RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
   RedisModule_Log(ctx, "notice", "redis version observed by redisearch : %d.%d.%d",
                   redisMajorVesion, redisMinorVesion, redisPatchVesion);
 
-  // Chain the config into RediSearch's global config
+  // Chain the config into RediSearch's global config and set the default values
+  clusterConfig = DEFAULT_CLUSTER_CONFIG;
```

Comment:
Can you explain why this is needed? It was not here before...

Reply:
seems to be a mistake and also the easiest way to maintain default values. see my [reply here](https://github.com/RediSearch/RediSearch/pull/3853#discussion_r1349698820)